src/components/Space-rockets.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  cancelReservationAction,
+  getRocketsAction,
+  setReserveAction,
+} from "../redux/Rockets";
+
+const Rockets = () => {
+  const { rockets } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!rockets) {
+      dispatch(getRocketsAction());
+    }
+  }, []);
+
+  const setReserve = (id) => {
+    dispatch(setReserveAction(id));
+  };
+
+  const cancelReserve = (id) => {
+    dispatch(cancelReservationAction(id));
+  };
+
+  return (
+    <section className="inner-padding">
+      {rockets &&
+        rockets.map((rocket) => (
+          <div className="rocket flex my-5" key={rocket.id}>
+            <div className="rocket-img w-2/6">
+              <img className="pl-6" src={rocket.flickr_images} alt="rocket" />
+            </div>
+            <div className="rocket-info pl-6">
+              <h1>{rocket.rocket_name}</h1>
+              <p>
+                {rocket.reserved && (
+                  <span className="reserve-badge">Reserved</span>
+                )}
+                {rocket.description}
+              </p>
+              <div className="button-div">
+                {!rocket.reserved ? (
+                  <button
+                    className="reserve-btn"
+                    onClick={() => setReserve(rocket.id)}
+                    type="button"
+                  >
+                    {" "}
+                    Reserve
+                  </button>
+                ) : (
+                  <button
+                    className="cancel-reserve-btn"
+                    onClick={() => cancelReserve(rocket.id)}
+                    type="button"
+                  >
+                    {!rocket.reserved ? "Reserve" : "Cancel Reserve"}
```

Comment:
- [ ] For this particular project, the React conditional render syntax is required for conditionally displaying both the Rockets and the Missions:

![image](https://user-images.githubusercontent.com/57421823/151388114-547dfc15-0380-44e6-b274-b7f7fbe510bb.png)

Don't worry! I'll show you how!! 💪 💪 

This is an example:

![image](https://user-images.githubusercontent.com/57421823/151388576-018db458-f6d7-4bfc-b0f9-a90dd7768f4f.png)

Another example with a kind of "if/else" scheme:

![image](https://user-images.githubusercontent.com/57421823/151396150-0479bcef-bb3b-44d2-b1a7-e696d222f7d2.png)


The rule is that if the condition on the left of the `&&` is true, the action of the right will be run. 💪 

Now that you know how to do it kindly please implement this syntax to your missions and rockets conditional rendering (i.e. for the "reserved" badge, the "join" badge, etc). ✔️